demo/features/0453-issue-credential.feature

```diff
@@ -64,9 +64,10 @@ Feature: RFC 0453 Aries agent issue credential
 
     @WalletType_Askar_AnonCreds @SwitchCredTypeTest
```

Comment:
Can we add an `@Release` tag here? So the tests run on our nightly and pre-release workflows.